docs/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+[EVG-<number>](https://jira.mongodb.org/browse/EVG-<number>)
+
+### Description 
+< add description, context, thought process, etc >
```

Comment:
I've always thought it would be nice if these subtitles were [proper comments](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments) so I don't have to make the effort to delete them 😄 

```suggestion
<!-- add description, context, thought process, etc -->
```